src/main/java/bio/terra/app/configuration/ResourceBufferServiceConfiguration.java

```diff
@@ -0,0 +1,69 @@
+package bio.terra.app.configuration;
+
+import com.google.auth.oauth2.AccessToken;
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.auth.oauth2.ServiceAccountCredentials;
+import com.google.common.collect.ImmutableList;
+import java.io.FileInputStream;
+import java.io.IOException;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+/** Configuration for managing connection to Buffer Service. * */
+@Configuration
+@EnableConfigurationProperties
+@ConfigurationProperties(prefix = "rbs")
+public class ResourceBufferServiceConfiguration {
+
+    private boolean enabled;
+    private String instanceUrl;
+    private String poolId;
+    private String clientCredentialFilePath;
+
+    //I think we'd want to re-use our app scopes.
+    private static final ImmutableList<String> BUFFER_SCOPES = ImmutableList.of("openid", "email", "profile");
```

Comment:
No need to use `ImmutableList` here. Using `List.of()` builds an immutable list, and its type is `List`.
```suggestion
    private static final List<String> BUFFER_SCOPES = List.of("openid", "email", "profile");
```